src/pages/PlaylistPage.js

```diff
@@ -33,7 +33,8 @@ const PlaylistPage = ({params, lessonsByLevel, coursePlaylists, t, showPlaylists
       </AutoAffix>
     </Col> : null;
   return (
-    <DocumentTitle title={capitalize(params.course) + ' | ' + t('title.codeclub')}>
+    <div>
+      <Head title={capitalize(params.course) + ' | ' + t('head.title')}/>
```

Comment:
Burde du ikke inkludert description også?

Reply:
Jo, men jeg tror det er best at jeg venter på PRen din. For jeg vil bruke filer i resources :)